scripts/README.md

```diff
@@ -67,8 +67,6 @@ The documentation must be updated on the `docs-vX` branch (`docs-v4` for `v4.X`
 
 Push the working branch on the documentation release branch to deploy the documentation with the latest changes.
 
-<!-- #default-branch-switch -->
```

Comment:
I think this should be before L74 and we still need to update the URL to: 

```diff
-https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v7
+https://app.netlify.com/sites/material-ui-x/deploys?filter=docs-v8
```

Reply:
oh, yes ... it being on this line felt like it points to the actual script